modules/caddyhttp/fileserver/browse.go

```diff
@@ -49,6 +49,8 @@ var BrowseTemplate string
 type Browse struct {
 	// Filename of the template to use instead of the embedded browse template.
 	TemplateFile string `json:"template_file,omitempty"`
+	// Determines whether or not targets of symlinks should be revealed.
+	RevealSymlink bool `json:"reveal_symlink,omitempty"`
```

Comment:
```suggestion
	RevealSymlinks bool `json:"reveal_symlinks,omitempty"`
```

Wondering if plural makes more sense :thinking: